File: C#/TopflytechCodec/TopflytechCodec/Entities/Rs232RfidMessage.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace TopflytechCodec.Entities
{
    class Rs232RfidMessage : Rs232DeviceMessage
    {
        private String rfid;

        public String Rfid
        {
            get { return rfid; }
            set { rfid = value; }
        }
    }
}

File: Java/topflytechCodec/src/com/topflytech/codec/entities/Rs232RfidMessage.java
package com.topflytech.codec.entities;
 
/**
 * Created by admin on 2017/11/27.
 */
public class Rs232RfidMessage extends Rs232DeviceMessage {
    private String rfid;

    public String getRfid() {
        return rfid;
    }

    public void setRfid(String rfid) {
        this.rfid = rfid;
    }
}
